README-template.md

```diff
@@ -62,13 +66,26 @@ This project is maintained by [Greenbone Networks GmbH](https://www.greenbone.ne
 
 ## Contributing
 
-Your contributions are highly appreciated. Please [create a pull request](https://github.com/greenbone/X/pulls) on GitHub. For bigger changes, please discuss it first in the [issues](https://github.com/greenbone/X/issues).
+Your contributions are highly appreciated. Please [create a pull request](https://github.com/greenbone/X/pulls) on GitHub. For bigger changes, please start a discussion with the development team via the [issues section at github](https://github.com/greenbone/X/issues) first.
```

Comment:
Starting a discussion is not enough. A consent has to be found. So "discuss it with the maintainers" would describe the process more precisely.

Reply:
Agreed, I have rephrased the sentence.